lightkurve/tests/test_targetpixelfile.py

```diff
@@ -18,8 +18,9 @@
 import astropy.units as u
 from astropy.utils.exceptions import AstropyWarning
 
-from ..targetpixelfile import KeplerTargetPixelFile, KeplerTargetPixelFileFactory
-from ..targetpixelfile import TessTargetPixelFile
+from ..targetpixelfile import KeplerTargetPixelFile, TargetPixelFileFactory
+from ..targetpixelfile import TessTargetPixelFile, TargetPixelFile
+from ..factory import TargetPixelFileFactory
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'TargetPixelFileFactory' from line 21](https://app.codacy.com/app/barentsen/lightkurve/pullRequest?prid=4287604)